pipelines/enterprise-contract.yaml

```
@@ -59,16 +59,41 @@ spec:
       type: string
       description: The name of the key in the ConfigMap that contains the CA bundle data.
       default: "ca-bundle.crt"
+    - name: REDUCE_SNAPSHOT
```

Perhaps (I'm not 100%), considering that folk might not know or care about what Application snapshot is:
```suggestion
    - name: SINGLE_COMPONENT
```

---

I renamed and refactored some of the parameters, task names and paths